source/dem/dem.cc

```diff
@@ -437,7 +439,11 @@ DEMSolver<dim>::particle_wall_contact_force()
 {
   // Particle-wall contact force
   pw_contact_force_object->calculate_pw_contact_force(
-    pw_pairs_in_contact, simulation_control->get_time_step(), momentum, force);
+    pw_pairs_in_contact,
+    simulation_control->get_time_step(),
+    momentum,
+    force,
+    parameters);
```

Comment:
Which parameters are those? Couldnt you pass them when you call the constructor of the object?

Reply:
I don't see how.